tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/types/globalvar/GlobalVarNegativeTest.java

```diff
@@ -84,7 +84,8 @@ public void testConfigurableModuleVarDeclNegative() {
         BAssertUtil.validateError(result, i++, "configurable variable cannot be declared with var", 20, 1);
         BAssertUtil.validateError(result, i++, "type of configurable variable must be anydata&readonly", 22, 22);
         BAssertUtil.validateError(result, i++, "type of configurable variable must be anydata&readonly", 24, 14);
-        BAssertUtil.validateError(result, i++, "invalid token 'configurable'", 32, 18);
+        BAssertUtil.validateError(result, i++, "missing close brace token", 31, 1);
+        BAssertUtil.validateError(result, i++, "invalid token '}'", 35, 1);
```

Comment:
since local configurable var declaration is the issue here, we should at least give `invalid token 'configurable'` or try to validate after parsing and give an error message like `configurable variable cannot be declared locally`

Reply:
+1.  Create an issue #28429